package.json

```diff
@@ -53,16 +53,16 @@
     "eslint": "^9.15.0",
     "eslint-import-resolver-typescript": "^3.6.3",
     "eslint-plugin-import": "^2.31.0",
-    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-prettier": "^5.2.1",
     "husky": "^8.0.0",
     "jasmine-core": "~5.4.0",
     "karma": "^6.4.4",
     "karma-chrome-launcher": "^3.2.0",
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-jasmine": "^5.1.0",
-    "karma-jasmine-html-reporter": "^1.7.0",
-    "prettier": "^3.0.0",
-    "pretty-quick": "^3.1.0",
+    "karma-jasmine-html-reporter": "~2.1.0",
```

Comment:
Is there a reason to switch these from `^` to `~`?

Reply:
No, I think I got them confused.